libs/shared/ui/src/lib/components/button/button.spec.tsx

```diff
@@ -2,8 +2,12 @@ import { renderWithProviders } from '@qovery/shared/util-tests'
 import { Button } from './button'
 
 describe('Button', () => {
-  it('should match snapshot', () => {
-    const { baseElement } = renderWithProviders(<Button type="button">Foobar</Button>)
+  it('should match snapshot with loading', () => {
```

Comment:
We should keep a test with the non-loading version

Reply:
https://github.com/Qovery/console/pull/995/commits/c89f10da11a622fb99100f65979c71dfb581d658